packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -27,35 +36,40 @@ const defaultValue = 'This is a caption';
 const groupIdentifier = 'CAPTION VARIANTS';
 
 <Caption>
-  {select(label, LANGUAGE_VARIANTS, defaultValue, groupIdentifier)}
+  {select(label, LANGUAGE_VARIANTS, LANGUAGE_VARIANTS.English, groupIdentifier).text}
 </Caption>
 ```
 
 ### inputProvider
 
-N.B. the number of elements in the array should equal the number of arguments taken
-by the render function.
-
 ```jsx
```

Comment:
Can we maybe add a simple example of this in a real story book file? Would make it alot easier to review the behaviour.

Maybe just a simple div with 2 pieces of text that use the input provider?

Reply:
👍 done